eslint.config.mjs

```diff
@@ -64,7 +64,7 @@ export default [
 
     rules: {
       'react-hooks/rules-of-hooks': 'error',
-      'react-hooks/exhaustive-deps': 'warn',
+      'react-hooks/exhaustive-deps': 'off',
```

Comment:
oh, no, with this approach u will avoid all rules related to dependencies

Reply:
restore it